smartgymapi/lib/validation/sport_scheme.py

```diff
@@ -1,8 +1,26 @@
-from marshmallow import Schema, fields
+from dateutil.parser import parse
+
+from marshmallow import Schema, fields, ValidationError, validates
 
 
 class SportScheduleSchema(Schema):
     id = fields.Str(dump_only=True)
     user_id = fields.UUID(required='User id is required')
     name = fields.Str(required='Name is required')
     reminder_minutes = fields.Integer(required='Reminder is required')
+    time = fields.Str(required='Time is required')
+    weekdays = fields.List(fields.Integer(), required='Weekday is required')
+    is_active = fields.Boolean(default=True)
+
+    @validates('time')
+    def validate_time(self, time_):
```

Comment:
Heb ik gecheckt maar alsnog werkte het niet, in aangezien het gewoon gevalideerd wordt is het wel prima toch?